miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/BulkCreateTableBackend.java

```diff
@@ -0,0 +1,38 @@
+package uk.ac.bbsrc.tgac.miso.webapp.util;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import org.springframework.ui.ModelMap;
+import org.springframework.web.servlet.ModelAndView;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+/**
+ * Create a Handsontable for propagating or editing a particular entity type
```

Comment:
There are 3 classes with the same description saying "propagating or editing," but one has "Propagate" in its name, and one has "Edit"